otx/mpa/seg/semisl/stage.py

```diff
@@ -11,17 +11,17 @@
 logger = get_logger()
 
 
-class SemiSegStage(SegStage):
+class SemiSLSegStage(SegStage):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
     def configure_task(self, cfg, training, **kwargs):
         """Adjust settings for task adaptation"""
         super().configure_task(cfg, training, **kwargs)
 
-        self.configure_semi(cfg, **kwargs)
+        self.configure_semisl(cfg, **kwargs)
 
-    def configure_semi(self, cfg, **kwargs):
+    def configure_semisl(self, cfg, **kwargs):
```

Comment:
Could this function be merged with `configure_task`?

Reply:
I hope to leave as is. configure_task is collections of configurations, so as other stages. It is good to seperate the method functionality as minimum.

Reply:
How about configure_data? Actually detection does

Reply:
I agree with jaeguk. The name `configure_semisl` doesn't sound right.

Reply:
I will just remove and merge it both in incremental and semisl. It would be good to seperate the function when more method or config is added and realized that the name configure_task covers all itself. Thanks for comments.